packages/lesswrong/lib/modules/utils/schemaUtils.js

```
@@ -33,6 +33,24 @@ export const generateIdResolverMulti = ({collectionName, fieldName}) => {
   }
 }
 
+const simplSchemaToGraphQLtype = (type) => {
```

This does feel like it should throw an error if it gets an invalid type